doc/latex.rst

```diff
@@ -1274,16 +1276,19 @@ forget the underscore separating the prefix from the property names.
   The defaults:
 
   * ``3pt`` for :rst:dir:`code-block` (since 6.0.0) and all corners,
-  * ``12pt`` for the bottom right corner of :dudir:`topic`, other corners are
-    straight,
+  * ``8pt`` for all corners of :dudir:`topic`,
+  * ``12pt`` for the bottom right corner of contents_, others use ``0pt``,
+  * ``12pt`` for the top-left and bottom-right corners for :dudir:`sidebar`,
+    ``0pt`` for top-right and bottom-left.
   * all radii set to ``5pt`` for :dudir:`note`, :dudir:`hint` and
     :dudir:`tip`,
   * ``0pt``, i.e. straight corners for all other directives.
 
-  .. versionchanged:: 7.4.0
+  .. versionchanged:: 7.4.0 :dudir:`topic` and :dudir:`note`\ -like
+     admonitions acquire (at least one) rounded corners.
 
-     :dudir:`topic` and :dudir:`note`\ -like admonitions acquire (at least one)
-     rounded corners.
+  .. versionchanged:: 8.1.0 Separate defaults for :dudir:`topic`, contents_,
+     and :dudir:`sidebar`.
```

Comment:
Ditto

Reply:
I will apply (lacking English word) return to start of line after 8.1.0 in one commit for all next